Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.8.6-slim-buster as base
+
+ENV POETRY_VERSION=1.1.0
+
+RUN pip install "poetry==$POETRY_VERSION"
+
+COPY pyproject.toml .
+RUN poetry config virtualenvs.create false
+RUN poetry install --no-interaction --no-ansi
+
+COPY . ./app
+WORKDIR /app
+
+FROM base as development
+EXPOSE 5000
+ENTRYPOINT ["./run_dev.sh"]
+
+FROM base as production
+RUN pip install gunicorn
```

Comment:
You should include gunicorn in the poetry dependencies (pyproject.toml) so that its version is source controlled.